dftimewolf/lib/processors/turbinia.py

```diff
@@ -161,10 +160,10 @@ def Process(self):
     """Process files with Turbinia."""
     log_file_path = os.path.join(self._output_path, 'turbinia.log')
     print('Turbinia log file: {0:s}'.format(log_file_path))
-
-    if self.state.input and not self.disk_name:
-      _, disk = self.state.input[0]
-      self.disk_name = disk.name
+    vm_containers = self.state.GetContainers(containers.ForensicsVM)
+    if vm_containers and not self.disk_name:
+      forensics_vm = vm_containers[0]
+      self.disk_name = forensics_vm.evidence_disk.name
```

Comment:
What if evidence_disk is None? I don't see anything enforcing that evidence_disk needs to be set, or have a .name attribute

Reply:
evidence_disk needs to be passed when creating the ForensicsVM container, the parameter doesn't default to None. There's no enforcment, but the object type that should be passed is a `libcloudforensics.providers.gcp.internal.GoogleComputeDisk`, and has a `name` attribute. In this case isn't it safe to assume that ForensicsVM containers will be correctly formed?